### ISSUE TITLE
Unused method parameters should be removed.`RSPEC-1172`

### ISSUE THREAD

**Issue by yeikel** [openrewrite/rewrite#1652]
I imagine that to "do not harm", we should start with `private` methods

https://rules.sonarsource.com/java/RSPEC-1172

**Comment by tkvangorder** [comment]
Yeah, I think be much easier to implement once data flow is available.
